configure

```diff
@@ -430,8 +430,8 @@ check_incdir() {
    for lua_h in \
       "$incdir/lua/$LUA_VERSION/lua.h" \
       "$incdir/lua$LUA_VERSION/lua.h" \
+      "$incdir/lua${LUA_VERSION/.//}/lua.h" \
```

Comment:
This is not POSIX sh compatible. See https://stackoverflow.com/a/21913014/282536
I recommend using `tr` for this case instead.

Reply:
yep, that was the shellcheck failure